dist/tools/firmware_metadata/README.md

```diff
@@ -0,0 +1,36 @@
+# Metadata generator for firmware verification
+This program generates a binary file containing a metadata structure as
```

Comment:
> Maybe add an empty line after title.

What this will change?

> uick question: does the file only contain the metadata or the metadata + the initial firmware ? If it's the second case, it should be more explicit. Otherwise, one can easily understand that the generated binary file only contain the struct.

It is the first case.